### ISSUE TITLE
Subql-starter’s docker-compose.yml  can't run project properly report

### ISSUE THREAD

**Issue by Index0011** [subquery/subql#1204]
The default configures of subquery can not run properly in one’s own env. When I create a new project on my own env, the default docker-compose.yml file can not start the project properly. Maybe we can update the relatedd subql config?
here is the error logs when using default docker-compose-yaml:
......
Pulling postgres       ... done
Pulling subquery-node  ... done
Pulling graphql-engine ... done
Starting phala-world-subquery_postgres_1 ... done
phala-world-subquery_subquery-node_1 is up-to-date

ERROR: for graphql-engine  Container "fbc5cb307514" is unhealthy.
ERROR: Encountered errors while bringing up the project.

**Comment by AtelyPham** [comment]
Hi team!

I'm facing the same error as above.

I'm trying to run SubQuery project locally (using `yarn start:docker`) and connect with my custom node (currently my node is running locally). But the SubQuery output said my `graphql-engine is unhealthy`. The log of my `subquery-node` container show _`connection failed`_ (I attached 2 screenshots below).

Do you have any idea to debug or fix it? Thank you 🙏

<img width="567" alt="CleanShot 2022-07-28 at 00 40 46@2x" src="https://user-images.githubusercontent.com/60747384/181313526-ec048c5c-5c69-439a-8694-7fe76e74513c.png">

<img width="567" alt="CleanShot 2022-07-28 at 00 41 14@2x" src="https://user-images.githubusercontent.com/60747384/181313628-8e6cb8e7-74bb-4756-ae5a-e67a3e1e5d27.png">

**Comment by AtelyPham** [comment]
Also found this issue https://github.com/subquery/subql/issues/189.

I updated my `docker-compose.yml` and `project.yaml`:

<details>
<summary>docker-compose.yml</summary>

```yml
version: '3'

services:
	  postgres:
		# ...
	
	  subquery-node:
		# ...
	    environment:
	      NETWORK_ENDPOINT: ws://${DOCKER_GATEWAY_HOST:-host.docker.internal}:9944

	  graphql-engine:
		# ...
	    environment:
	      NETWORK_ENDPOINT: ws://${DOCKER_GATEWAY_HOST:-host.docker.internal}:9944
		
```

</details>

<details>
<summary>project.yaml</summary>

```yaml
network:
	endpoint: ${NETWORK_ENDPOINT}

```

</details>

But the behaviour is still the same, _`connection_failed`_.

**Comment by AtelyPham** [comment]
Dup https://github.com/subquery/subql/issues/1220

**Comment by stwiname** [comment]
This should be fixed with 1.6.1
